src/Database/Database.php

```diff
@@ -1645,6 +1645,21 @@ public function processlist()
 		return (["list" => $statelist, "amount" => $processes]);
 	}
 
+	/**
+	 * Fetch a database variable
+	 *
+	 * @param string $name
+	 * @return string content
+	 */
+	public function getVariable(string $name)
+	{
+		$result = $this->fetchFirst("SHOW GLOBAL VARIABLES WHERE `Variable_name` = ?", $name);
+		if (!isset($result['Value'])) {
+			return null;
+		}
+		return $result['Value'];
```

Comment:
Equivalent and shorter:
```suggestion
		return $result['Value'] ?? null;
```